product-service/src/functions/getProductsList/handler.ts

```diff
@@ -0,0 +1,34 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
+// import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+
+// import schema from './schema';
+
+import { Product } from '../../model/product';
+import productsData from '../../store/products.json';
+const getProductsList = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
```

Comment:
is this the same that `getProductsAvailableList`?

Reply:
No, the function getProductsAvailableList returns a filtered array with products whose stock count is greater than 0